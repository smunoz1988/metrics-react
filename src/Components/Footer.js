import '../styles/footer.css';

const Footer = () => (
  <footer>
    Created by Santiago MuÃ±oz
  </footer>
);

export default Footer;
