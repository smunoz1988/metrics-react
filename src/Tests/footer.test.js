import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import Footer from '../components/Footer';

describe('Footer', () => {
  test('renders the footer text correctly', () => {
    render(<Footer />);
    const footerElement = screen.getByText('Created by Santiago Mu√±oz');
    expect(footerElement).toBeInTheDocument();

    expect(footerElement).toMatchSnapshot();
  });
});
