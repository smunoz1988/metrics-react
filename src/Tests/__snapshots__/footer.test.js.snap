// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders the footer text correctly 1`] = `
<footer>
  Created by Santiago Mu√±oz
</footer>
`;
